### PR TITLE
Do not include invalid file in symfony53 set

### LINKED ISSUE #364: Seems to be broken in rector 0.15.5?
The file being included, added in https://github.com/rectorphp/rector-symfony/pull/358/files#diff-f480b78b4fb61aa7087e89ccf55c40fff94c5111b885a08626c3b4382c0b936cR27, does not exist.

### PR DESCRIPTION
**Pull request by marmichalski**
Closes #364.

### CODE REVIEW

File: config/sets/symfony/symfony53.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->sets([SymfonySetList::ANNOTATIONS_TO_ATTRIBUTES]);
 
-    $rectorConfig->import(__DIR__ . '/symfony53-types.php');
+    $rectorConfig->import(__DIR__ . '/symfony50-types.php');
```
Comment (marmichalski) [inline config/sets/symfony/symfony53.php:27]: I am not entirely sure if this should be even included as part of this set?

Comment (TomasVotruba) [reply config/sets/symfony/symfony53.php:27]: Not at all, it should be removed.

I'm surprised PHPStan missed this.

File: config/sets/symfony/symfony53.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
```
Comment (marmichalski) [inline config/sets/symfony/symfony53.php:28]:
```suggestion
```

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**